### PR TITLE
[SEDB-55] Styler Update Profile Completed

### PR DESCRIPTION
**Pull request by jameskanCa**

### CODE REVIEW

File: client/src/redux/actions/creators/inquiryActionCreators.js [hunk 1]
```diff
@@ -66,28 +65,7 @@ export const GetInquiries = (userId) => {
 export const CreateInquiry = (userId, post) => {
   return async (dispatch) => {
     try {
-      // // TODO change to API call
```
Comment (tsangela) [inline client/src/redux/actions/creators/inquiryActionCreators.js:69]: Nice 🎉

File: client/src/components/UserProfileForm/StylerProfileEditForm.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  const [affiliations, setAffiliations] = useState(props.userInfo.affiliations);
+  const [biography, setBiography] = useState(props.userInfo.biography);
+
+  const onSubmit = () => {
+    const { userInfo } = props;
+    const profile = {
+      name: userInfo.name,
+      gender: userInfo.gender,
+      // TODO FIX THE DATE CAUSES AN ERROR
+      birthdate: userInfo.birthdate,
```
Comment (tsangela) [inline client/src/components/UserProfileForm/StylerProfileEditForm.js:23]: we can remove the un-editable fields
```javascript
      name: userInfo.name,
      gender: userInfo.gender,
      // TODO FIX THE DATE CAUSES AN ERROR
      birthdate: userInfo.birthdate,
```

in `userReducer`, change to return:
```javascript
    case userActionTypes.UPDATE_PROFILE:
      // todo(atsang): make api call to update db
      return {
        isLoggedIn: true,
        userType: state.userType,
        profile: {
          ...state.profile,
          ...action.profile,
        },
      };
```

pulled down to test

Comment (tsangela) [reply client/src/components/UserProfileForm/StylerProfileEditForm.js:23]: also change in `StyleeProfileEditForm`

File: client/src/components/UserProfileForm/StylerProfileEditForm.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+const SelectWidth100 = { width: '100%' };
+
+export default function StyleeProfileEditForm(props) {
+  const dispatch = useDispatch();
+  const [location, setLocation] = useState(props.userInfo.location);
+  const [affiliations, setAffiliations] = useState(props.userInfo.affiliations);
+  const [biography, setBiography] = useState(props.userInfo.biography);
+
+  const onSubmit = () => {
+    const { userInfo } = props;
```
Comment (tsangela) [inline client/src/components/UserProfileForm/StylerProfileEditForm.js:18]: then `userInfo` doesn't need to be passed as props (unused)

File: client/src/components/UserProfileForm/StylerProfileEditForm.js (hunk 2 above)
Comment (jameskanCa) [reply client/src/components/UserProfileForm/StylerProfileEditForm.js:23]: Is the reducer change already present in ur PR?

Comment (jameskanCa) [reply client/src/components/UserProfileForm/StylerProfileEditForm.js:23]: Just checked it isn't i'll put it in.

File: client/src/redux/reducers/userReducer.js [hunk 4]
```diff
@@ -39,17 +39,19 @@ const userReducer = (state = initialState, action) => {
         ? state
         : {
             isLoggedIn: true, // todo(atsang): assumption + hacky
-            userType: userTypes.STYLEE, // todo(atsang): stub
+            userType: userTypes.STYLE, // todo(atsang): stub
```
Comment (tsangela) [inline client/src/redux/reducers/userReducer.js:42]: typo?

Comment (jameskanCa) [reply client/src/redux/reducers/userReducer.js]: ya

**Pull request merged**